include/triton/Dialect/TritonNvidiaGPU/IR/TritonNvidiaGPUOps.td

```diff
@@ -208,6 +208,43 @@ def TTNG_WaitBarrierOp : TTNG_Op<"wait_barrier", [
     let assemblyFormat = "$alloc `,` $phase attr-dict (`,` $pred^)? (`deps` $deps^)? `:` qualified(type($alloc)) (`,` type($deps)^)?";
 }
 
+def TTNG_ArriveBarrierOp : TTNG_Op<"arrive_barrier", [DeclareOpInterfaceMethods<MemoryEffectsOpInterface>]> {
+  let summary = "perform the arrive operation on an mbarrier";
+  let description = [{
+    The `ttng.arrive_barrier` operation performs the "arrive" operation on an
+    mbarrier object in shared memory. The operation requires a `count` attribute
+    of at least 1, and increases the pending arrival count of the mbarrier by
```

Comment:
Shouldn't it be "decreases"?

Reply:
Yes you're right. It should be decreasing